packages/react-core/src/operations/groupByDate.d.ts

```diff
@@ -0,0 +1,9 @@
+import { GroupByFeature } from '../types'
+import { GroupDateTypes } from './GroupDateTypes';
+
+export function groupValuesByDateColumn(
+  data: [], // TODO: geojson types
```

Comment:
remove TODO comment

Reply:
![image](https://user-images.githubusercontent.com/9151432/135580687-7d290fe9-2bb1-4e19-8d35-b99d74c9134c.png)
Everywhere?

Reply:
👍🏻  I did it